docs/source/api.rst

```diff
@@ -16,7 +16,7 @@ Custom exceptions
 -----------------
 
 .. warning:: For any of these errors to work, you need to register the error handlers in your Flask app.
-    To do this, you can call :meth:`flask_utils.errors.register_error_handlers` with your Flask app as an argument.
+    To do this, you can call :meth:`register_error_handlers` with your Flask app as an argument.
```

Comment:
This seem to have broken the link in the documentation preview:
https://flask-utils--29.org.readthedocs.build/en/29/api.html#custom-exceptions

Reply:
I am a bit confused as to what broke in this one, since in the latest readthedocs build there doesn't seem to be a link here either.
https://flask-utils.readthedocs.io/en/latest/api.html#custom-exceptions

Reply:
Oh you're right, that's weird. There should be one though

Reply:
I cant seem to find a reference to `register_error_handlers` in the docs, only to `_register_error_handlers`, maybe thats why? Because it should be generating the link yeah.

Reply:
Or maybe its deprecated? At least that's what I got from reading https://flask-utils--29.org.readthedocs.build/en/29/api.html#flask_utils.errors._register_error_handlers

Reply:
No you are correct, I didn't rename the function properly in my last update. It is a private function now
```suggestion
    To do this, you can call :meth:`register_error_handlers` with your Flask app as an argument.
```
```suggestion
    To do this, you can call :meth:`_register_error_handlers` with your Flask app as an argument.
```

Reply:
But should It stay like that? Or should I rewrite it to instruct the user to pass the `register_error_handlers=True` argument to the Flaskutils class

Reply:
You're correct that would be better, thank you ! :D

Reply:
Done!